bcl2fastq/handlers/bcl2fastq_handlers.py

```diff
@@ -112,7 +113,7 @@ def create_config_from_request(self, runfolder, request_body):
         runfolder_input = "{0}/{1}".format(runfolder_base_path, runfolder)
 
         if not os.path.isdir(runfolder_input):
-            raise RuntimeError("No such file: {0}".format(runfolder_input))
+            raise ArteriaException("No such file: {0}".format(runfolder_input))
```

Comment:
Weird since my local tests have been passing and I guess this should have triggered an import error... :confused: